src/pages/invite/InviteError.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div :class="getClass">
+    <Card class="content">
+      <Paragraph>You have not been invited to this workspace.</Paragraph>
+      <Button @click="logout">switch account</Button>
+      <Break />
+      <Small>
+        currently logged in as:
+        <Break />
+        {{ user.name }} ({{ user.email }})
+      </Small>
+    </Card>
+  </div>
+</template>
+
+<script>
+import Component from '/components/sharedScripts/component'
+import gql from 'graphql-tag'
+
+export default new Component({
+  name: 'InviteError',
+  apollo: {
+    user: {
+      query: gql`
+        {
+          user {
+            name
+            callname
+            email
+            id
+          }
+        }
+      `,
+      error(err) {
+        if (
+          err.message.replace('GraphQL error:', '').trim() === 'NOT_LOGGED_IN'
```

Comment:
I've seen most of this code before... Can it be moved to the custom Error class? :thinking: